lua/cfc_random_spawn/module/sv_player_spawning.lua

```diff
@@ -197,13 +216,74 @@ local function getPlayerPopularityFromPoint( point, plys, radiusSqr )
     return totalScore
 end
 
+local function getDynamicPvpCenter( measurablePlayers )
+    local playersAveragePos = Vector( 0, 0, 0 )
+    for _, ply in ipairs( measurablePlayers ) do
+        playersAveragePos = playersAveragePos + ply:GetPos()
```

Comment:
theres already a getPlyAvg function used other places in the code. May be better to use that.

The getPlyAvg function itself could also probably be improved. It could not take count manually and use the `#` operator.

Reply:
getplyavg used count instead of #, to enable some specific behaviour iirc, something about picking the best pvpcenter when there's multiple

Reply:
That functionality wasn't as important as i thought. Applied this suggestion!